src/components/presentations/PasswordReset/ForgotPassword.jsx

```diff
@@ -35,25 +36,29 @@ class ForgotPassword extends FormComponent {
    */
   render() {
     const { isLoadingReducer, user } = this.props;
+    const { userProfile } = user;
     const { loader } = isLoadingReducer;
-    if (user) return <Redirect to="/" />;
+    if (Object.keys(userProfile).length) return <Redirect to="/" />;
     return (
-      <div className="forgot-password">
-        {loader && <Loader />}
-        <React.Fragment>
-          <div className="form-wrap">
-            <div className="form-cont">
-              <Form unstackable onSubmit={this.handelClick}>
-                <p>FORGOT PASSWORD</p>
-                {this.renderInput('email', 'Email', 'email')}
-                <Button className="submit" type="submit">
-                  Reset my password
-                </Button>
-              </Form>
+      <React.Fragment>
+        <NavBar />
+        <div className="forgot-password">
+          {loader && <Loader />}
+          <React.Fragment>
+            <div className="form-wrap">
+              <div className="form-cont">
+                <Form unstackable onSubmit={this.handelClick}>
+                  <p>FORGOT PASSWORD</p>
+                  {this.renderInput('email', 'Email', 'email')}
```

Comment:
`this.renderInput('email', 'Email', 'email')` - The arguments passed to this function should be named constants to improve readability @Akinmyde